packages/SketchMorph2-Core.package/M2CustomizableColorMorph.class/instance/initialize.st

```diff
@@ -0,0 +1,9 @@
+initialization
+initialize
+	super initialize.
+	self
+		model: [self offColor = Color transparent ifTrue:
+			[| userColor | userColor := self selectedColor.
```

Comment:
temporary variable definition and initialisation probably shouldn't be on the same line